x/farming/keeper/genesis_test.go

```diff
@@ -1,6 +1,8 @@
 package keeper_test
 
 import (
+	_ "github.com/stretchr/testify/suite"
```

Comment:
How about moving `_` imports to the bottom of the import statement, like:
```go
import (
        sdk "github.com/cosmos/cosmos-sdk/types"

        "github.com/tendermint/farming/x/farming/types"

        _ "github.com/stretchr/testify/suite"
)
```

Reply:
That is fair enough. I'll move that import at the bottom.